nemo/collections/tts/torch/data.py

```diff
@@ -79,6 +79,10 @@ def __init__(
         min_duration: Optional[float] = None,
         ignore_file: Optional[Union[str, Path]] = None,
         trim: bool = False,
+        trim_ref: Optional[float] = None,
```

Comment:
@rlangman see here, `trim_ref` is an optional float type argument. We can override it in the YAML config. Given current use cases NeMo supported, it may be good to constrain it with a float number. We can extend it with a callable func in future if there is new use case available.

Reply:
Yea, you would only need to be able to override it with a float (not another numpy function).